src/helpers/Api.js

```diff
@@ -2,7 +2,7 @@ import axios from 'axios';
 import { MY_NAMESPACE_LABELS, MY_NAMESPACE_PLAYLIST_EDIT, MY_NAMESPACE_PLAYLISTS } from './consts';
 
 const API_BACKEND             = process.env.REACT_APP_API_BACKEND;
-const ASSETS_BACKEND          = process.env.REACT_APP_ASSETS_BACKEND;
+const ASSETS_BACKEND          = 'https://kabbalahmedia.info/assets/';//process.env.REACT_APP_ASSETS_BACKEND;
```

Comment:
You should use here process.env, right?

Reply:
yes, will change it for prod, I cant deploy asserts backend for staging